libs/blocks/marquee-anchors/marquee-anchors.css

```diff
@@ -0,0 +1,175 @@
+/*
+[marquee-anchors]
+variants: (dark, transparent)
+*/
```

Comment:
meant to keep this comment?

Reply:
I scrapped this unnecessary comment above and fixed/added a few, ex.  //variants:dark comments next to their class position.